crates/config/src/lib.rs

```diff
@@ -2140,9 +2140,20 @@ impl Config {
     /// This normalizes the default `evm_version` if a `solc` was provided in the config.
     ///
     /// See also <https://github.com/foundry-rs/foundry/issues/7014>
+    #[expect(clippy::disallowed_macros)]
```

Comment:
```suggestion
    #[expect(clippy::disallowed_macros)]
```

This should not be necessary

Reply:
Nice catch, fixed! @zerosnacks